lib/matplotlib/axis.py

```diff
@@ -1760,6 +1760,11 @@ def get_minpos(self):
         raise NotImplementedError()
 
 
+def tick_space_heuristic(axis_length, label_fontsize, label_fontsize_factor):
```

Comment:
This should be private I think.

Reply:
You mean prepend the function with an underscore?

Reply:
Yes.